### DISCUSSION TITLE
I installed tweepy but it the code returns "ModuleNotFoundError"

### DISCUSSION THREAD

**Discussion by baadr634** [tweepy/tweepy#1812] [Questions] [answered]
I installed a tweepy to use in my code using the following command: pip install tweepy, it installed perfectly like any other module I installed before the only problem is when I write: import tweepy at the top of my coding file, the code doesn't function and produces this error: 
`ModuleNotFoundError: No module named 'tweepy'
`
**Notes:**

1- I checked the lib directory and the module exists there
2- I tried uninstalling and reinstalling the module a couple of times and the issue isn't resolved
3- I tried installing it using pip3, same issue
4- I'm using Microsoft Visual Studio Code.

**Comment by Harmon758** [comment, marked as answer]
Make sure you're installing Tweepy in the same [environment](https://code.visualstudio.com/docs/python/environments) that you're running your code in.

**Reply by baadr634** [reply]
I'm pretty sure I am in the right directory, I see the other modules that I use there :c

**Reply by Harmon758** [reply]
If you're running it through VSC, do you have the right interpreter selected?

**Reply by baadr634** [reply]
ermmm, can you tell me how i can check?

**Reply by Harmon758** [reply]
The VSC documentation I linked covers it.
